x-pack/plugins/enterprise_search/server/lib/indices/text_analysis.ts

```diff
@@ -0,0 +1,298 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+interface LanguageDataEntry {
+  name: string;
+  stemmer: string;
+  stop_words: string;
+  custom_filter_definitions?: object;
+  prepended_filters?: string[];
+  postpended_filters?: string[];
+}
+
+const LanguageData: Record<string, LanguageDataEntry> = {
```

Comment:
question: Why are these UpperCamelCase rather than lowerCamelCase?

Reply:
Good catch! These were all uppercase constants in Ruby (`LANGUAGE_DATA`), and in my indecision of how to handle them in Typescript I PascalCased it (I just learned that word from the Kibana linter) rather than camelCased. I'll switch it to camelCase.